File: spec/code_samples/Php_http2/boards@hidden_entries/delete.php
<?php

$client = new http\Client;
$request = new http\Client\Request;

$request->setRequestUrl('https://www.directfreight.com/api/boards/hidden_entries');
$request->setRequestMethod('DELETE');
$request->setQuery(new http\QueryString(array(
  'entry_id' => 'SOME_STRING_VALUE'
)));

$client->enqueue($request)->send();
$response = $client->getResponse();

echo $response->getBody();

File: spec/code_samples/Perl_perl5/boards@hidden_entries/delete.pl
#!/usr/bin/perl
use Modern::Perl;
use Data::Dumper;
use HTTP::Request::JSON;
use LWP::UserAgent::JSON;
my $request = HTTP::Request::JSON->new("DELETE",
"'https://www.directfreight.com/api/boards/hidden_entries?entry_id=SOME_STRING_VALUE'");
my $browser = LWP::UserAgent::JSON->new();
my $response = $browser->request($request);
print Dumper($response->json_content);